nc-news/src/components/Articles.jsx

```diff
@@ -0,0 +1,48 @@
+import { useState, useEffect } from "react";
+import axios from "axios";
+import ArticleCard from "./ArticleCard";
+import ArticleNavBar from "./ArticleNavBar";
+
+function Articles({ topic }) {
+  const [loading, setLoading] = useState(true);
+  const [articles, setArticles] = useState([]);
+
+  let request = "";
+
+  if (topic === undefined) {
+    request = `https://be-nc-news-zmuo.onrender.com/api/articles`;
+  } else {
+    request = `https://be-nc-news-zmuo.onrender.com/api/articles?topic=${topic}`;
+  }
```

Comment:
Because of the wonders of ✨`axios`✨ you can pass the query as a second argument to your `.get`, if it's undefined it will be excluded so it handles this conditional logic on your behalf 

Here's an example in the [notes](https://notes.northcoders.com/courses/js-front-end/axios#axios-params)